lib/methwhy.g

```diff
@@ -90,13 +90,89 @@ end);
 ##      also for each not applicable method all reasons why it is not
 ##      applicable.
 ##  </Item>
-##  <Mark>6</Mark>
-##  <Item>
-##      also the function body of the selected method(s)
-##  </Item>
 ##  </List>
 ##  <P/>
-##  When a method returned by <Ref Func="ApplicableMethod"/> is called then
+##  Since <C>ApplicableMethod</C> returns a function, <C>Print(last);</C> 
+##  may be used to view the code. 
```

Comment:
Sure, but if I want to see the code for *all* the methods, then this is not a substitute. E.g. if I do this: `ApplicableMethod(ViewObj, [ "test" ], 6, 10);` then it'll print several functions.